app/Resources/views/themes/ezplatform/form/search.html.twig

```diff
@@ -0,0 +1,10 @@
+{{ form_start(searchForm) }}
```

Comment:
I would recommend to use `searchBundleForm`, maybe at some point we will introduce a global `search` on all the website.